tests/unit/services/cookies-test.js

```diff
@@ -74,6 +74,11 @@ describe('CookiesService', function() {
         expect(afterRoundtrip).to.eq(value);
       });
 
+      it('handles invalid values for cookies', function() {
+        document.cookie = '=blank';
+        expect(this.subject().read('blank')).to.be.undefined;
```

Comment:
`blank` would actually be the value here. I think this should be `expect(this.subject().read('')).to.be.undefined;` instead.